modules/OAuth2Clients/language/en_us.lang.php

```diff
@@ -42,16 +42,34 @@
     die('Not A Valid Entry Point');
 }
 
-$mod_strings = array (
+$mod_strings = [
```

Comment:
This change is OK for Crowdin translation platform.
But is there a point to change the format and having one additional string format into the system?
I hoped things would get cleaner in the future!

Reply:
I think the short array syntax **is** the cleaner form. 

It is supported since PHP 5.4, so in all currently supported PHP versions, and it is the de facto standard for most modern PHP applications.

While it is not explicitly mentioned in PSR2, all the examples given in [the PSR code guide](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md) do use short syntax as well.

Reply:
no problem for me, the thing is, this should be done for all language files at the same time so there is no different code formats.

Reply:
I'd love to do just that :+1: